annual_dual_failure_probability.py

```
@@ -0,0 +1,111 @@
+            'joint_mtbf_hours': hours between expected simultaneous failures,
+            'annual_probability': annual chance of simultaneous failure (fraction),
+            'annual_probability_percent': annual chance in percent
+        }
+    """
+    joint_failure_rate_per_hour = 1 / system_mtbf * (rto_hours / system_mtbf)
+    joint_mtbf_hours = 1 / joint_failure_rate_per_hour
+    annual_probability = joint_failure_rate_per_hour * HOURS_PER_YEAR
+    return joint_mtbf_hours, annual_probability
+
+
+def fit_to_mtbf(fit):
```

Ruff (0.12.2)

Missing return type annotation for public function `fit_to_mtbf`

(ANN201)